### DISCUSSION TITLE
Une compensation pour les badges recu en doublon

### DISCUSSION THREAD

**Discussion by BananePlantain** [Crownicles/Crownicles#2448] [Suggestions fermées]
Salut,
ici je pense aux gds qui peuvent donner un badge lors qu'un membre  (ou plus) ne l'as pas et du coup les autres qui l'ont déjà :sob: rien,voili voilou (et ca,ca devrait pas partir en maxi débat/concours d'égo/chasse au sorcière 👀 enfin normalement)

**Comment by BastLast** [comment]
c'est fait exprés :)
